packages/floating-label/README.md

```diff
@@ -0,0 +1,84 @@
+# mwc-floating-label
```

Comment:
This and some of the other smaller/internal package READMEs seem to be in the old style -- update them? Doesn't necessarily need screenshots or the detailed API docs, though.